docs/examples/safe-remote.rst

```diff
@@ -106,6 +106,7 @@ you can use state machine-like constructs inside a contract.
         /// is called.
         function confirmPurchase()
             external
+            onlyBuyer
```

Comment:
It's only ``confirmPurchase`` here that sets ``buyer`` in the first place (see the second line in the body), so this change is invalid: after the change it'd be impossible to confirm a purchase.